src/ImageRunModal.jsx

```diff
@@ -1101,18 +1195,8 @@
                            Dialogs.close();
                        }
                    }}
-                   title={this.props.pod ? cockpit.format(_("Create container in $0"), this.props.pod.Name) : _("Create container")}
-                   footer={<>
-                       <Button variant='primary' id="create-image-create-run-btn" onClick={() => this.onCreateClicked(true)} isDisabled={!image && selectedImage === ""}>
-                           {_("Create and run")}
-                       </Button>
-                       <Button variant='secondary' id="create-image-create-btn" onClick={() => this.onCreateClicked(false)} isDisabled={!image && selectedImage === ""}>
-                           {_("Create")}
-                       </Button>
-                       <Button variant='link' className='btn-cancel' onClick={Dialogs.close}>
-                           {_("Cancel")}
-                       </Button>
-                   </>}
+                   title={this.props.pod ? cockpit.format(_("Create container in $0"), this.props.pod.Name) : (this.props.prefill ? _("Clone container") : _("Create container"))}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230428-111750-01fd85db-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)